src/views/Pools/components/BountyModal.tsx

```diff
@@ -75,9 +80,18 @@ const BountyModal: React.FC<BountyModalProps> = ({
       <Flex alignItems="flex-start" justifyContent="space-between">
         <Text>{t("You'll claim")}</Text>
         <Flex flexDirection="column">
-          <Text bold>{cakeBountyToDisplay} CAKE</Text>
+          <Balance bold value={cakeBountyToDisplay} decimals={7} unit=" CAKE" />
           <Text fontSize="12px" color="textSubtle">
-            ~ {dollarBountyToDisplay} USD
+            <>
+              ~
+              <InlineBalance
+                fontSize="12px"
+                color="textSubtle"
+                value={dollarBountyToDisplay}
+                decimals={2}
+                unit=" USD"
```

Comment:
Maybe we should consider adding a `prefix` prop to this component. Just a thought.

Reply:
Fuck yea we should

Reply:
Added this in Hachi, great shout